tasks/task02/area_of_triangle.py

```diff
@@ -0,0 +1,37 @@
+import math
+
+
+def calculate_area_basement_height():
+    input_array = input("Enter base and height: ").split(" ")
+    height = int(input_array[0])
+    base = int(input_array[1])
+    return (height * base) / 2
+
+
+def calculate_area_two_sides_angle():
+    input_array = input("Enter two sides and angle between them: ").split(" ")
```

Comment:
it's only positive cases, when user set only correct value, but you need try to prevent incorrect values.

An example of incorrect values: negative values, only one number, zero etc.
And this is to relate second function.